runtime/near-wallet-contract/wallet-contract/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+//! Temporary implementation of the Wallet Contract.
+//! See https://github.com/near/NEPs/issues/518.
+//! Must not use in production!
+// TODO(eth-implicit) Change to a real Wallet Contract implementation.
+
+use hex;
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::{env, near_bindgen, AccountId, Promise};
+use rlp::Rlp;
+
+#[near_bindgen]
+#[derive(Default, BorshDeserialize, BorshSerialize)]
+pub struct WalletContract {}
+
+#[near_bindgen]
+impl WalletContract {
+    /// For the sake of this placeholder implementation, we assume simplified version of the `rlp_transaction`
+    /// that only has 3 values: `To`, `Value`, and `PublicKey`. We assume this is a transfer transaction.
+    /// The real implementation would obtain the public key from `Signature`.
+    pub fn execute_rlp(&self, target: AccountId, rlp_transaction: Vec<u8>) {
+        let rlp = Rlp::new(&rlp_transaction);
+
+        let to: String = match rlp.val_at(0) {
```

Comment:
Is there some sort of schema library that we could use to automatically deserialize the fields for us? 

Doesn't need to be in this PR, just generally speaking it would be nice to separate the deserializing from the business logic validation and then the execution.